### DISCUSSION TITLE
Map misalignment bug

### DISCUSSION THREAD

**Discussion by beyoung** [mapsforge/mapsforge#1479] [Q&A] [answered]
When I created maps by thoes steps , I encouter map misalignment  bug 
<img width="1421" alt="image" src="https://github.com/mapsforge/mapsforge/assets/2298389/d7fbb640-741b-4de3-a512-8c9aa0b80732">

* cutting tile pbf from planet-240415.osm.pbf using command  `osmium extract -b -123.75,40.979898,-112.5,48.922499 ./planet-240415.osm.pbf -o ./02123.osm.pbf --overwrite --set-bounds`
* convert pbf to map using command `./osmosis-0.49.2/bin/osmosis --rb file=./02123.osm.pbf --mapfile-writer file=02123.map type=ram tag-conf-file=./osmosis/default-tag.xml zoom-interval-conf=14,0,16`

Then I converted all 02123.osm.pbf lines to shapefile (It's toooo slow to open pbf in qgis), the command is `ogr2ogr output-all-lines.shp 02123-n.osm.pbf lines`   In QGIS it looks like
<img width="1840" alt="image" src="https://github.com/mapsforge/mapsforge/assets/2298389/cd4d9b06-d362-4873-96fc-745d9649eda8">

Then I tried to convert idaho pbf(downloaded directly from https://download.geofabrik.de/)
(because location 42.56711,-115.86362 in idaho) to map , it looks like 
<img width="1421" alt="image" src="https://github.com/mapsforge/mapsforge/assets/2298389/76a802e9-80ef-4541-b12f-abfc14b7088e">

I dont know why the misalignment maps, and it's only happens to part of maps(in USA) ,   it's normal  in asia/ europe etc.
<img width="922" alt="image" src="https://github.com/mapsforge/mapsforge/assets/2298389/0da061a1-4f86-4d44-ac25-56b374884be6">
below are some normal maps
asia
<img width="1421" alt="image" src="https://github.com/mapsforge/mapsforge/assets/2298389/d3d2d773-982f-4cae-a5ca-9c79a9fbeba4">
europe
<img width="1421" alt="image" src="https://github.com/mapsforge/mapsforge/assets/2298389/085ac897-9606-4fb6-803c-028dda0c6fa4">

version
- osmosis version 0.49.2
- mapsforge-map-writer version 0.21.0
- osmium version 1.13.1
- libosmium version 2.16.0

**Comment by devemux86** [comment, marked as answer]
Please see the detailed [map creation guide](https://github.com/mapsforge/mapsforge/blob/master/docs/MapCreation.md).

For an automated process, see the [mapsforge-creator](https://github.com/mapsforge/mapsforge-creator) using Geofabrik and the old [mapsforge-mapcreator](https://github.com/mapsforge/mapsforge-mapcreator) using planet.osm.

**Reply by beyoung** [reply]
Thank you for your reply, I want to know if the osmium-extracted map creation workflow not work well ?  and I should abandon it  .
